### PR TITLE
Fix #2551 No search result found for still persist on screen

### LINKED ISSUE #2551: "No search result found for" still persists on screen.
**Actual Behaviour**
If we exclude all the local folders and then search for any random folder, "No search result found for" persists on screen.Even if we include all those excluded folders the above text of no search result is still there over the images

**Expected Behaviour**
The text of "No search result found for" should not be always there on screen

**Steps to reproduce it**
1.Select and Exclude all the local folders.
2.Search for a random folder and press back button
3.The text of No search result found for is still there
4.Now remove all those folders from exclude, the text of no search result found is displayed over all the images.

**Screenshots of the issue**
image
image

**Would you like to work on the issue?**
 Yes

### PR DESCRIPTION
**Pull request by theabhishekavi**
Fixed #2551 

Changes: The default text of search view found on screen is removed.

Screenshots before the change:-
image
image

Screenshots after the change:
image

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 1]
```diff
@@ -1700,10 +1700,14 @@ private void showsnackbar(Boolean result) {
...
     }
 
-    private void checkNoSearchResults(String result){
-        textView.setText(getString(R.string.null_search_result) + " " + '"' + result + '"' );
-        textView.setTextColor(getTextColor());
-        textView.setVisibility(View.VISIBLE);
+    private void checkNoSearchResults(String result,int length){
+        if (length>0) {
+            textView.setText(getString(R.string.null_search_result) + " " + '"' + result + '"');
+            textView.setTextColor(getTextColor());
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: Just hide the textview

Comment (theabhishekavi) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: So basically you want to remove the textview from screen?

Comment (theabhishekavi) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: or you are asking me to implement View.Gone?

Comment (abishekvashok) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: View.gone

**Comment by theabhishekavi** [comment]
@abishekvashok @sauravvishal8797 please review

**Pull request merged**